martin-mbtiles/src/tile_copier.rs

```diff
@@ -267,18 +279,40 @@ mod tests {
         ));
     }
 
+    #[actix_rt::test]
+    async fn copy_force_simple() {
+        let src_filepath = PathBuf::from("../tests/fixtures/files/world_cities.mbtiles");
```

Comment:
May want to call them same as other tests for consistency

Reply:
I meant you can call them `src` and `dst` just like other tests, but this is is just a nit